### DISCUSSION TITLE
Facing an issue while downloading go packages from the private repo.

### DISCUSSION THREAD

**Discussion by rajat2911** [renovatebot/renovate#28923] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

Hi Team 
I am facing an issues where  I got to see the dependency updates in my repo for mockgen package and it needs to download some packages from my another repo but its failing , can anyone help me on the same 

```
go: downloading github.com/xxxxx/xxxxx-services v1.11.0
../attribute.go:5:2: reading github.com/xxxx/xxxx/go.mod at revision v1.11.0: git ls-remote -q origin in /go/pkg/mod/cache/vcs/xxxxxxx: exit status 128:
	fatal: could not read Username for 'https://github.com': terminal prompts disabled
Confirm the import path was entered correctly.
If this is a private repository, see https://golang.org/doc/faq#git_https for additional information.
go: downloading github.com/xxxxxx/xxxxxv1.11.0
../attribute.go:5:2: reading github.com/xxxxx/xxxxxx-services/go.mod at revision v1.11.0: git ls-remote -q origin in /go/pkg/mod/cache/xxxxx/xxxxx: exit status 128:
	fatal: could not read Username for 'https://github.com': terminal prompts disabled
Confirm the import path was entered correctly.```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Try Google searching for `"go get" "terminal prompts disabled"`

**Reply by rajat2911** [reply]
So you mean to say I should use go env -w GOPRIVATE=https://github.com/<<repo name>> when running the post upgrade commands ?

**Reply by rarkins** [reply]
I don't know for sure what the problem or solution is, but I do know that it's not a Renovate-specific problem so the solution is to be found elsewhere. If you conclude that it's a problem which Renovate could automatically work around for others in future, please let us know

**Reply by rajat2911** [reply]
Yes , When running renovate in a self hosted runner it fetches the packages globally but returns an error when fetching the packages from private repo , I thought might be setting some host rules for GitHub repo would fix this

**Reply by rarkins** [reply]
https://docs.renovatebot.com/getting-started/private-packages/#gomod

If you log with debug log level then you can see if Renovate is configuring such env
